packages/react-components/src/components/MessageThread.tsx

```diff
@@ -1239,107 +1233,126 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
   );
 
   const theme = useTheme();
-  const chatMessageRenderStyles = useChatMessageRenderStyles();
-
-  const messagesToDisplay = useMemo(
-    () =>
-      memoizeAllMessages((memoizedMessageFn) => {
-        return messages.map((message: Message, index: number): JSX.Element => {
-          let key: string | undefined = message.messageId;
-          let statusToRender: MessageStatus | undefined = undefined;
-
-          if (
-            message.messageType === 'chat' ||
-            /* @conditional-compile-remove(data-loss-prevention) */ message.messageType === 'blocked'
-          ) {
-            if ((!message.messageId || message.messageId === '') && 'clientMessageId' in message) {
-              key = message.clientMessageId;
-            }
-            if (showMessageStatus && message.mine) {
-              switch (message.messageId) {
-                case lastSeenChatMessage: {
-                  statusToRender = 'seen';
-                  break;
-                }
-                case lastSendingChatMessage: {
-                  statusToRender = 'sending';
-                  break;
-                }
-                case lastDeliveredChatMessage: {
-                  statusToRender = 'delivered';
-                  break;
+
```

Comment:
Looks like `showMessageDate` isn't used anymore in in 

![image](https://github.com/Azure/communication-ui-library/assets/98852890/0802582d-9bf0-4521-a4de-48b6ab03cc8e)